<?xml version="1.0"?>


<launch>

  
  <node name="gimbal_yaw" pkg="pid" ns="gimbal_yaw" type="controller" respawn="true" >
      <param name="Kp" value="-0.02" />
      <param name="Ki" value="-0.076" />
      <param name="Kd" value="-0.002" />
      <param name="upper_limit" value="10" />
      <param name="lower_limit" value="-10" />
      <param name="windup_limit" value="4" />
      <param name="max_loop_frequency" value="100.0" />
      <param name="min_loop_frequency" value="100.0" />
    </node>
  
  <node name="gimbal_pitch" pkg="pid" ns="gimbal_pitch" type="controller" respawn="true" >
      <param name="Kp" value="0.02" />
      <param name="Ki" value="0.076" />
      <param name="Kd" value="0.002" />
      <param name="upper_limit" value="10" />
      <param name="lower_limit" value="-10" />
      <param name="windup_limit" value="4" />
      <param name="max_loop_frequency" value="100.0" />
      <param name="min_loop_frequency" value="100.0" />
    </node>





</launch>
